src/core/i18n/utils.js

```diff
@@ -181,6 +181,16 @@ export function makeMomentLocale(locale) {
   return locale.replace('_', '-').toLowerCase();
 }
 
+// Functionality based on oneLine form declandewet/common-tags https://goo.gl/4PzaJI
+// If this function is altered muffinresearch/babel-gettext-extractor also needs
```

Comment:
Nitpick but I think you should link to the URL of the repo here, not just the name. Makes it easier to scan and a bit more obvious (I did a double-take when reading it!)